parsedatetime/tests/TestRussianLocale.py

```diff
@@ -78,6 +78,24 @@ def testDates(self):
         self.assertExpectedResult(self.cal.parse('25.8', start), (target, 1))
         self.assertExpectedResult(self.cal.parse('25.08', start), (target, 1))
 
+    def testDatesLang(self):
+        if sys.version_info >= (3, 0):
+            target = datetime.datetime(2006, 8, 25, 23, 5).timetuple()
```

Comment:
@WarmongeR1 can you file an issue about the PY2k error and reference this conversation so we can know it needs fixing